emr_run_spark.py

```diff
@@ -89,7 +92,11 @@ def _create_steps(job_flow_name=None,
     subprocess.check_call('aws s3 cp {} {}'.format(local_zip_file, zip_file_on_s3), shell=True)
     zip_file_on_host = '{}/{}'.format(sources_on_host, zip_file)
     spark_main_on_host = '{}/{}'.format(sources_on_host, spark_main)
-    spark_main_args = spark_main_args.split() if spark_main_args else ['']
+    # spark_main_args = spark_main_args.split() if spark_main_args else ['']
+    packages_to_add = []
+    if use_mysql:
+        packages_to_add.append('mysql:mysql-connector-java:5.1.39')
```

Comment:
probably a more generic way to achieve this is to add a parameter `--packages` that might take `mysql:mysql-connector-java:5.1.39` as an argument (and more arguments if more packages are required). I mean, now that you don't need all these stupid workarounds to get the jar in place, that's doable.